src/components/RunList/index.tsx

```diff
@@ -5,187 +5,258 @@ import type {
   SetQuery,
 } from "use-query-params";
 import { useDebounce } from "usehooks-ts";
-import type {
-  GridFilterModel,
-  GridRowClassNameParams,
-  GridValueFormatterParams,
-  GridValueGetterParams,
-  GridRenderCellParams,
-  GridColDef,
-} from "@mui/x-data-grid";
+import {
+  useMaterialReactTable,
+  MaterialReactTable,
+  type MRT_ColumnDef,
+  type MRT_PaginationState,
+  type MRT_Updater,
+  type MRT_ColumnFiltersState,
+} from 'material-react-table';
+import { type Theme } from "@mui/material/styles";
+import { parse } from "date-fns";
 
 import { useRuns } from "../../lib/paddles";
-import { formatDate, formatDuration } from "../../lib/utils";
-import DataGrid from "../DataGrid";
+import { formatDate, formatDay, formatDuration } from "../../lib/utils";
 import IconLink from "../../components/IconLink";
+import type { Run } from "../../lib/paddles.d";
+import { RunStatuses } from "../../lib/paddles.d";
+import useDefaultTableOptions from "../../lib/table";
 
-function resultsGetter(params: GridValueGetterParams) {
-  return params.row.results[params.field];
-}
 
-const columns: GridColDef[] = [
+const DEFAULT_PAGE_SIZE = 25;
+const NON_FILTER_PARAMS = [
+  "page",
+  "pageSize",
+];
+
+const columns: MRT_ColumnDef<Run>[] = [
   {
-    field: "user",
+    header: "status",
```

Comment:
If we hide a column we can't filter by it, so we have to keep status visible.

Let's talk more about the suite column; I felt like the utility it provided wasn't necessarily worth increasing load times on every pageload, but I could be convinced :)

Reply:
1. Makes sense! 
2. Let's leave the suite column field as text, if select-menu is a popular feature that people miss then we can change that later.
To battle increased load time, we can set react-query's  `staleTime` and `cacheTime` (renamed as `gcTime` in v5) to `Infinity`: https://tanstack.com/query/v4/docs/react/reference/useQuery
In fact we can do that for `useMachineTypes` too since that looks pretty constant query too!